pkg/odo/cli/pipelines/bootstrap.go

```diff
@@ -63,6 +65,20 @@ func (io *BootstrapParameters) Validate() error {
 	if len(removeEmptyStrings(strings.Split(gr.Path, "/"))) != 2 {
 		return fmt.Errorf("repo must be org/repo: %s", strings.Trim(gr.Path, ".git"))
 	}
+	if io.GitOpsWebhookSecret == "" {
+		gitopsSecret, err := GenerateSecureString()
```

Comment:
gitOpsSecret as variable name
generateSecureString as func name (i.e. I don't think it nneds to be exported.)

Reply:
I will change this.

Reply:
We've discussed this, and I think we can move it to `pkg/utils/secrets.go` with `pkg/utils/secrets_test.go`

The test should generate a fixed number of items, add them to `map[string]bool` and at the end, check that the `len()` of that map is the same as the fixed number.

Oh, and the test should also iterate over all the strings returned, and check that they're all len() == 20.

Reply:
Test case with the requirements mentioned above completed.